OntologyMapper/src/README.md

```diff
@@ -109,17 +111,29 @@ A collection of mappings from one or more property names to another property nam
   ],
   "PropertyProjections": [
     {
-      "OutputDtmiFilter": "*",
+      "OutputDtmiFilter": ".*",
```

Comment:
With this change do the preexisting mappings need to be updated with this PR to continue to function?

Reply:
I'll do a follow on PR to update the existing mappings files. Need to update this nuget version first, and then do the update

Reply:
Lets chat about this going forward offline, approving for now